homeassistant/components/point/__init__.py

```diff
@@ -172,10 +172,12 @@ async def handle_webhook(hass, webhook_id, request):
 class MinutPointClient:
     """Get the latest data and update the states."""
 
-    def __init__(self, hass: HomeAssistant, config_entry: ConfigEntry, session):
+    def __init__(
+        self, hass: HomeAssistant, config_entry: ConfigEntry, session: PointSession
+    ) -> None:
         """Initialize the Minut data object."""
-        self._known_devices = set()
-        self._known_homes = set()
+        self._known_devices: set = set()
+        self._known_homes: set = set()
```

Comment:
@fredrike, as `point` codeowner could you confirm if there is a better hint we could use here for `_known_devices` and `_known_homes`?

Reply:
I'm not sure what you are looking for but those two are sets with string identifiers of one or more homes and devices

Reply:
If device_id is always a string, then we can use `set[str]` which is better than an untyped set `set`.
Does this suggestion look ok?
```suggestion
        self._known_devices: set[str] = set()
        self._known_homes: set[str] = set()
```

Reply:
I'll have to Verify but it looks Ok.